contrib/init/bitcoind.service

```diff
@@ -1,22 +1,25 @@
+# It is not recommended to modify this file in-place, because it will
+# be overwritten during package upgrades. If you want to add further
+# options or overwrite existing ones then use
+# $ systemctl edit bitcoind.service
+# See "man systemd.service" for details.
+
+# Note that almost all daemon options could be specified in
+# /etc/bitcoin/bitcoin.conf
+
 [Unit]
-Description=Bitcoin's distributed currency daemon
+Description=Bitcoin daemon
 After=network.target
 
 [Service]
+ExecStart=/usr/bin/bitcoind -daemon -conf=/etc/bitcoin/bitcoin.conf -pid=/run/bitcoind/bitcoind.pid
```

Comment:
Probably should use `/etc/bitcoind` to be consistent with `/run/bitcoind` or `/run/bitcoin` to be consistent with `/etc/bitcoind`

Reply:
If I'm setting up systemd do I need to create the bitcoin user and group myself?